### PR TITLE
Enzyme: force rerun for registration

### PR DESCRIPTION
**Pull request by wsmoses**
@vchuravy let me know if you know a better way of retriggering, per CI hitting reg rate limit apparently: https://buildkite.com/julialang/yggdrasil/builds/11549#01905cfd-bded-4180-9bb4-f4210d91cb98

### CODE REVIEW

**Comment by wsmoses** [comment]
@giordano if you have ways to force the retrigger without an entire rebuild

**Comment by giordano** [comment]
Yes, I'll do it.

**Comment by giordano** [comment]
Upload failed again after restarting :confused:

**Comment by giordano** [comment]
I'm not sure reopening this PRs helps though, there's a problem with the upload process :confused:  CC: @staticfloat

**Comment by wsmoses** [comment]
yeah in bg gonna try something to see if it reduces the size of the tarball

File: E/Enzyme/build_tarballs.jl [hunk 1]
```diff
@@ -102,7 +102,8 @@ fi
 echo ${CMAKE_FLAGS[@]}
 cmake -B build -S enzyme -GNinja ${CMAKE_FLAGS[@]}
 
-ninja -C build -j ${nproc} install
+ninja -C build -j ${nproc} libEnzyme libEnzymeBCLoad
+cp build/libEnzyme*.{so,dylib,dll} ${prefix}
```
Comment (giordano) [inline E/Enzyme/build_tarballs.jl:106]:
```suggestion
install -Dvm 755 "build/libEnzyme.${dlext}" "${libdir}/libEnzyme.${dlext}"
```

File: E/Enzyme/build_tarballs.jl [hunk 2]
```diff
@@ -142,7 +143,7 @@ for llvm_version in llvm_versions, llvm_assertions in (false, true)
     if llvm_version >= v"17" && llvm_assertions
         # Windows is broken for LLVM16_jll see https://github.com/JuliaPackaging/Yggdrasil/pull/8017#issuecomment-1930838052
         filter!(p -> !Sys.isapple(p), platforms)
-    end
+    end
```
Comment (giordano) [inline E/Enzyme/build_tarballs.jl:146]:
```suggestion
    end
```

**Comment by wsmoses** [comment]
@giordano @staticfloat @vchuravy this still fails on registration.

Is there a way to do this manually in the bg?

**Pull request merged**